test/run_test

```diff
@@ -597,19 +598,19 @@ run_s2i_test() {
   s2i usage --pull-policy=never "${IMAGE_NAME}" &>/dev/null
 
   echo "  Testing s2i build"
-  s2i build "file://${test_dir}/test-app" "${IMAGE_NAME}" "${IMAGE_NAME}-testapp"
-  local image_name_backup="${IMAGE_NAME}"
-  export IMAGE_NAME="${IMAGE_NAME}-testapp"
 
-  local container_name=s2i_config_build
-  _s2i_test_image "s2i_config_build" ""
+  local s2i_image_name=$IMAGE_NAME-testapp_$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 10 | head -n 1)
+  ct_path_append images_to_clean "$s2i_image_name"
 
-  # return back original value for IMAGE_NAME
-  export IMAGE_NAME="${image_name_backup}"
+  s2i build "file://${test_dir}/test-app" "${IMAGE_NAME}" "$s2i_image_name"
+  IMAGE_NAME=$s2i_image_name _s2i_test_image s2i_config_build ""
 
   echo "  Testing s2i mount"
-  cp -Lr "${test_dir}/test-app" "${s2i_test_app_dir}/"
-  _s2i_test_image "_s2i_test_mount" "-v ${s2i_test_app_dir}/test-app:/opt/app-root/src/:z"
+  create_volume_dir
+  cp -Lr "$test_dir/test-app/postgresql-init" "$volume_dir"
+  setfacl -R -m u:26:r-x "$volume_dir" # Execute bit is needed only for dirs
+
+  IMAGE_NAME=$s2i_image_name _s2i_test_image _s2i_test_mount "-v ${volume_dir}:/opt/app-root/src/:z"
```

Comment:
Shouldn't this run with the original (un-extended) image?

Reply:
Oh, right, good catch (not a new bug, though?)

Reply:
From what I can see the old version ran it that way:
```
local image_name_backup="${IMAGE_NAME}
export IMAGE_NAME="${IMAGE_NAME}-testapp"
...
export IMAGE_NAME="${image_name_backup}"
echo "  Testing s2i mount"
cp -Lr "${test_dir}/test-app" "${s2i_test_app_dir}/"
_s2i_test_image "_s2i_test_mount" "-v ${s2i_test_app_dir}/test-app:/opt/app-root/src/:z"
```